### PR TITLE
Create jsconfig.json file

### PR DESCRIPTION
**Pull request by YanaMIvanova**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | No
| BC breaks?    | No

This PR introduces `jsconfig.json` file, which enables code editors such as `VS Code` to use webpack aliases

### CODE REVIEW

**Comment by Elorfin** [comment]
This should not be commited. This is a proprietary file for VSCode.

**Comment by maieutiquer** [comment]
> This should not be commited. This is a proprietary file for VSCode.
> 
> EDIT :
> I just discovered it and only found references to `VSCode`, but can be interesting if it's supported by other editors.
> It works with PHPStorm.
> 
> We usually don't put editors config in the code base (we use PHPStorm and we have created a separate repo for its config).

I understand and we also don't commit proprietary files, but this seemed almost like an emerging standard that seemed supported by most used editors and adding that file might have made it easier for other developers to start contributing as well

Other options might be to add a "Editor setup" in the docs (or Readme), or creating a separate repo as you mentioned (although this might be overkill for such a small file), but it might be just easier to include it here, let us know what you think we should do

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "baseUrl": ".",
+    "module": "esnext",
+    "checkJs": false,
+    "jsx": "react",
+    "allowSyntheticDefaultImports": true,
+    "target": "esnext",
+    "moduleResolution": "node",
+    "paths": {
+    "#/main/app/*": ["./src/main/app/Resources/modules/*"]
+    }
```
Comment (maieutiquer) [inline jsconfig.json:14]: Indentation:

```suggestion
    "paths": {
      "#/main/app/*": ["./src/main/app/Resources/modules/*"]
    }
```

**Comment by Elorfin** [comment]
It misses too many things to be interesting to introduce it for now. I'll consider pushing mine once it will be more complete.

**Pull request closed without merging**